spec/unit/pops/parser/parse_application_spec.rb

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/pops'
+
+# relative to this spec file (./) does not work as this file is loaded by rspec
+require File.join(File.dirname(__FILE__), '/parser_rspec_helper')
```

Comment:
Ok .. serves me right for mindlessly cutting and pasting from other tests ;) Fixed.